e2e/runner/run_cypress_local.ts

```diff
@@ -39,67 +34,46 @@ if (options.MB_EDITION === "ee" && missingTokens.length > 0) {
 }
 
 printBold(`Running Cypress with options:
-  - CYPRESS_TESTING_TYPE : ${options.CYPRESS_TESTING_TYPE}
   - MB_EDITION           : ${options.MB_EDITION}
-  - BACKEND_PORT         : ${options.BACKEND_PORT}
-  - BUILD_BACKEND        : ${options.BUILD_BACKEND}
+  - CYPRESS_TESTING_TYPE : ${options.CYPRESS_TESTING_TYPE}
+  - CYPRESS_GUI          : ${options.CYPRESS_GUI}
   - GENERATE_SNAPSHOTS   : ${options.GENERATE_SNAPSHOTS}
-  - OPEN_UI              : ${options.OPEN_UI}
 `);
 
 const init = async () => {
   const cliArguments = process.argv.slice(2);
   const userOverrides = await parseArguments(cliArguments);
 
+  const { isBackendRunning } = CypressBackend;
+
   printBold("⏳ Starting containers");
   shell("docker compose -f ./e2e/test/scenarios/docker-compose.yml up -d");
 
-  if (options.BUILD_BACKEND) {
```

Comment:
I ❤️LOVE❤️ the fact that the default is now running from source, but I do not like removing the ability to build/run from a jar. I think this would be pretty easy to keep - which also facilitates preserving our CI testing setup

Reply:
We want to decouple the cypress runner from the environment. If you want to setup a funky backend however you want, go for it. But the runner isn't going to be the place where you do it anymore

Reply:
running metabase from a .jar is not a "funky backend" it's the way we ship metabase, the way we test it in CI, and we should support doing it locally with a single command

Reply:
I cannot stress enough the importance of the decoupling. I absolutely agree we should have an easy way to build, or build + run the JAR, but that *should not* be in the same command that spins cypress up. Why is this so controversial?

Both Cypress and the backend/metabase read from the same env var MB_JETTY_PORT. That is their connection. Everything else is just noise.